doc/admx/en-US/DesktopAppInstaller.adml

```diff
@@ -37,6 +37,14 @@ If you disable this setting, users will not be able to install packages with loc
 If you enable or do not configure this policy, users will be able to enable the ability override the SHA256 security validation in the Windows Package Manager settings.
 
 If you disable this policy, users will not be able to enable the ability override the SHA256 security validation in the Windows Package Manager settings.</string>
+      <string id="EnableIgnoreMalwareScan">Enable App Installer Malware Scan Override</string>
+      <string id="EnableIgnoreMalwareScanExplanation">
+        This policy controls whether or not the Windows Package Manager can be configured to enable the ability to ignore the result of the malware scan perfomed on archive type packages
+
+        If you enable or do not configure this policy, users will be able to bypass the Archive Malware Scan security validation using command line arguments.

```

Comment:
```suggestion
        If you enable or do not configure this policy, users will be able to enable the ability to override the Archive Malware Scan security validation using command line arguments.
        
        If you disable this policy, users will not be able to enable the ability to override the Archive Malware Scan security validation using command line arguments.

```

nit: Just to be consistent with OverrideHashMismatch above, and it's not actually "bypassing" the step

Reply:
Committed